Libraries/polyfills/console.js

```diff
@@ -537,6 +543,7 @@ if (global.nativeLoggingHook) {
     table: consoleTablePolyfill,
     group: consoleGroupPolyfill,
     groupEnd: consoleGroupEndPolyfill,
+    groupCollapsed: consoleGroupCollapsedPolyfill,
```

Comment:
I think this should be the `consoleGroupPolyfill`. I believe `console.groupCollapsed` should behave the same as `console.group` in a non-debugger environment logging to the OS.

Reply:
Hey @newyankeecodeshop if I use `consoleGroupPolyfill` then the log won't be collapsed because of `GROUP_OPEN` inside `consoleGroupPolyfill`.

Reply:
@Eyesonly88 I don't understand how the `GROUP_OPEN` character vs. `GROUP_CLOSE` makes a difference about whether the group is shown collapsed. When you attach the Safari debugger to the JSContext, is it still using the console polyfill?

Reply:
@newyankeecodeshop Yes it is using the console polyfill. Without my code, there is no `groupCollapsed`. As you can see in the demo, using `GROUP_CLOSE` makes it collapsed.

Reply:
Exactly.